controller/control.go

```diff
@@ -1207,25 +1217,17 @@ func (c *Controller) IsReplicaRW(replicaInController *types.Replica) error {
 }
 
 // DeleteSnapshot ...
-func (c *Controller) DeleteSnapshot(replicas []types.Replica) error {
+func (c *Controller) DeleteSnapshot(snapshot string, replicas []types.Replica) error {
```

Comment:
is DeleteSnapshot triggered by controller? its supposed to be by replica right

Reply:
This function is not being used, will be removed. From controller snapshot deletion can be triggered manually via cli.